### PR TITLE
Docs/consistency

### PR DESCRIPTION
**Pull request by Starkteetje**
Fixes company/product names and some typos, adds missing EOF line breaks

### CODE REVIEW

File: docs/features/alerting.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ during runtime into the payload:
 Referring to any of these variables in the templates works by Jinja2 notation
 (e.g. `{{ timestamp }}`). You can update your payload dynamically by adding payload
 fields in `yaml` representation in the `payload_fields` key which will be translated
-to JSON by Helm as is. If your REST endpoint requires particular headers, you can
+to JSON by Helm as is. If your REST endpoint requires particular headers, your can
```
Comment (xopham) [inline docs/features/alerting.md:73]: that is an unwanted change, right?

File: requirements.txt [hunk 2]
```diff
@@ -7,4 +7,4 @@ parsedatetime~=2.6
 pytz~=2020.1
 python-dateutil~=2.8.1
 PyYAML~=5.4.1
-aiohttp~=3.7.4.post0
\ No newline at end of file
+aiohttp~=3.7.4.post0
```
Comment (xopham) [inline requirements.txt:10]: ?

File: tests/validators/cosign/test_cosign_validator.py [hunk 3]
```diff
@@ -62,7 +62,7 @@
 
 cosign_stderr_at_success = """
 The following checks were performed on each of these signatures:
-  - The cosign claims were validated
+  - The Cosign claims were validated
```
Comment (xopham) [inline tests/validators/cosign/test_cosign_validator.py:65]: that is cosign output and should stay as is

**Review by xopham — COMMENTED** [review]
I am very worried that these automated changes will break things we don't notic

File: requirements.txt (hunk 2 above)
Comment (Starkteetje) [reply requirements.txt:10]: Dunno why Github doesnt show it, but this added a linebreak at the end that wasn't present before
```sh
od -b requirements.txt 
0000000 106 154 141 163 153 176 075 061 056 061 056 062 012 162 145 161
0000020 165 145 163 164 163 176 075 062 056 062 064 056 060 012 162 146
0000040 143 063 063 063 071 055 166 141 154 151 144 141 164 157 162 176
0000060 075 060 056 061 056 062 012 145 143 144 163 141 176 075 060 056
0000100 061 065 012 152 163 157 156 163 143 150 145 155 141 176 075 063
0000120 056 062 056 060 012 160 141 162 163 145 144 141 164 145 164 151
0000140 155 145 176 075 062 056 066 012 160 171 164 172 176 075 062 060
0000160 062 060 056 061 012 160 171 164 150 157 156 055 144 141 164 145
0000200 165 164 151 154 176 075 062 056 070 056 061 012 120 171 131 101
0000220 115 114 176 075 065 056 064 056 061 012 141 151 157 150 164 164
0000240 160 176 075 063 056 067 056 064 056 160 157 163 164 060
0000256
```
vs 
```sh
od -b requirements.txt 
0000000 106 154 141 163 153 176 075 061 056 061 056 062 012 162 145 161
0000020 165 145 163 164 163 176 075 062 056 062 064 056 060 012 162 146
0000040 143 063 063 063 071 055 166 141 154 151 144 141 164 157 162 176
0000060 075 060 056 061 056 062 012 145 143 144 163 141 176 075 060 056
0000100 061 065 012 152 163 157 156 163 143 150 145 155 141 176 075 063
0000120 056 062 056 060 012 160 141 162 163 145 144 141 164 145 164 151
0000140 155 145 176 075 062 056 066 012 160 171 164 172 176 075 062 060
0000160 062 060 056 061 012 160 171 164 150 157 156 055 144 141 164 145
0000200 165 164 151 154 176 075 062 056 070 056 061 012 120 171 131 101
0000220 115 114 176 075 065 056 064 056 061 012 141 151 157 150 164 164
0000240 160 176 075 063 056 067 056 064 056 160 157 163 164 060 012
0000257
```

File: tests/data/config/err1.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (annekebr) [inline tests/data/config/err1.yaml:1]: What is this?

Comment (Starkteetje) [reply tests/data/config/err1.yaml]: Artifact of adding a line break to files that didn't end with one. Guess for empty file that's fine. Reverted it here

**Pull request merged**